lib/components/artist/artworks/grid.js

```diff
@@ -1,283 +0,0 @@
-/* @flow */
```

Comment:
so this is the `artist /artworks/grid` which is an unused dupe of `artwork_grids/infinite_grid_view.js`